src/main/paradox/docs/tutorial/knowledge-graph/thinking-in-graph.md

```diff
@@ -25,10 +25,10 @@ for a company (Eric --works for--> EPFL).
 [ontology][1]. This is a formal description that both humans
 and computers can understand and process. The metadata is said to be self descriptive.
 
- In the example below, you can see that the entity **Bob** is a **Person** (also known as the **type** of the entity)
-and he has a property **was born on** for this the value is **14 July 1990**.
+ In the example below, you can see that the entity **Toy Story** is an **Animated movie** (also known as the **type** of the entity)
+and he has a property **Release date** for this the value is **March 29th, 1995**.
 
- ![Graph Structure](./assets/kg-graph_structure.png)
+ ![Graph Structure](./assets/kg-structure0.png)
```

Comment:
the image `./assets/kg-structure0.png` is not present in the PR, I'm not sure if it was already present before or you forgot to add it to the commit?